### PR TITLE
Upgrade ductile version 0.4.4 -> 0.4.5

### PR DESCRIPTION
**Pull request by DeLaGuardo**
> Related https://github.com/advthreat/iroh/issues/7056

Version 0.4.5 fixes encoding for the bulk insert operation. Those changes should fix bundle-import related issues when non-ASCII symbols got replaced with `?`.

<a name="qa">[§](#qa)</a> QA
============================

1. Import bundle with entities containing non-ASCII characters somewhere in the description.
2. Fetch that entity
3. Non-ASCII symbols should not be replaced with `?` or altered in any other way.

<a name="release-notes">[§](#release-notes)</a> Release Notes
=============================================================

```
public: entities fields can have non-ASCII symbols
```

<a name="squashed-commits">[§](#squashed-commits)</a> Squashed Commits
======================================================================

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -158,6 +158,58 @@ jobs:
         # poor man's travis_retry
         run: for i in {1..${ACTIONS_RETRY_TIMES}}; do lein warm-ci-deps && break; done
       - run: ./scripts/check-dependabot
+
+  test-encoding:
```
Comment (ereteog) [inline .github/workflows/pr.yml:162]: @frenchy64 I let you approve this part :-)

**Review by ereteog — APPROVED** [review]
LGTM

**Review by frenchy64 — APPROVED** [review]
Nice, the test selectors live [here](https://github.com/threatgrid/ctia/blob/master/dev-resources/circleci_test/config.clj), they could also be updated in the same way as ductile's to exclude these tests if that's the intention.

**Comment by DeLaGuardo** [comment]
> Nice, the test selectors live [here](https://github.com/threatgrid/ctia/blob/master/dev-resources/circleci_test/config.clj), they could also be updated in the same way as ductile's to exclude these tests if that's the intention.

Thanks, but I don't think encoding tests should be disabled via test selectors for "normal" test runs. They are speedy even if we run them twice.

**Comment by frenchy64** [comment]
@DeLaGuardo Ah, I was more thinking the problem would be that `with-profile +test-encoding` is not enabled in normal runs.

**Comment by skriths** [comment]
Verified Incident creation and updating with non-ASCII symbols in TEST for release 1.108

**Pull request merged**